commit 3f58ed8df33891413db0216c93002ca005bcf0e5
Author: Andy Whitcroft <apw@canonical.com>
Date:   Mon Mar 31 16:57:11 2014 +0100

    UBUNTU: SAUCE: nouveau: missing outputs should be warnings
    
    We emit a number of messages on outputs which are not connected, when
    these are not fatal.  These are more appropriatly warnings.
    
    BugLink: http://bugs.launchpad.net/bugs/1300244
    Signed-off-by: Andy Whitcroft <apw@canonical.com>
    [ saf: Update for 4.14-rc1 ]
    Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/drivers/gpu/drm/nouveau/nouveau_bios.c b/drivers/gpu/drm/nouveau/nouveau_bios.c
index dd6fba55ad5d..f56599f842ac 100644
--- a/drivers/gpu/drm/nouveau/nouveau_bios.c
+++ b/drivers/gpu/drm/nouveau/nouveau_bios.c
@@ -935,7 +935,7 @@ static int parse_bit_tmds_tbl_entry(struct drm_device *dev, struct nvbios *bios,
 
 	tmdstableptr = ROM16(bios->data[bitentry->offset]);
 	if (!tmdstableptr) {
-		NV_ERROR(drm, "Pointer to TMDS table invalid\n");
+		NV_WARN(drm, "Pointer to TMDS table invalid\n");
 		return -EINVAL;
 	}
 
